openwisp_monitoring/monitoring/base/models.py

```diff
@@ -355,6 +355,21 @@ def write(
                 'send_alert': send_alert,
             }
             options['metric_pk'] = self.pk
+
+            # check if alert_on_related_field is present in metric configuration
+            if 'alert_on_related_field' in self.config_dict:
```

Comment:
maybe we can call this `alert_field`, then we can define that:
if `alert_field` is not supplied, the system assumes it's "field_name".

I would then create a method for this property, which behaves as described above, eg:

```python
@property
def alert_field(self):
    return self.config_dict.get('alert_field', self.field_name)

@property
def alert_on_related_field(self):
    return self.alert_field != self.field_name
```

```suggestion
            if alert_on_related_field:
                # etc.
```